tests/import_test.go

```diff
@@ -184,6 +186,110 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 
 })
 
+var _ = Describe("DataVolume Garbage Collection", func() {
+	var (
+		f        = framework.NewFramework(namespacePrefix)
+		ns       string
+		err      error
+		config   *cdiv1.CDIConfig
+		origSpec *cdiv1.CDIConfigSpec
+	)
+
+	BeforeEach(func() {
+		ns = f.Namespace.Name
+		config, err = f.CdiClient.CdiV1beta1().CDIConfigs().Get(context.TODO(), common.ConfigName, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		origSpec = config.Spec.DeepCopy()
+	})
+
+	AfterEach(func() {
+		By("Restoring CDIConfig to original state")
+		err = utils.UpdateCDIConfig(f.CrClient, func(config *cdiv1.CDIConfigSpec) {
+			origSpec.DeepCopyInto(config)
+		})
+		Expect(err).ToNot(HaveOccurred())
+		Eventually(func() bool {
+			config, err = f.CdiClient.CdiV1beta1().CDIConfigs().Get(context.TODO(), common.ConfigName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			return reflect.DeepEqual(config.Spec, *origSpec)
+		}, 30*time.Second, time.Second).Should(BeTrue())
+	})
+
+	It("[test_id:XXXX] Should garbage collect dv after completion when CDIConfig DataVolumeTTLSeconds is set", func() {
+		By("Verify DataVolumeTTLSeconds is not set by default")
+		Expect(config.Spec.DataVolumeTTLSeconds).To(BeNil())
+
+		By("Set DataVolumeTTLSeconds to 0")
+		ttl := int32(0)
+		err = utils.UpdateCDIConfig(f.CrClient, func(config *cdiv1.CDIConfigSpec) {
+			config.DataVolumeTTLSeconds = &ttl
+		})
+		Expect(err).ToNot(HaveOccurred())
+
+		dv := utils.NewDataVolumeWithHTTPImport("gc-test", "100Mi", fmt.Sprintf(utils.TinyCoreIsoURL, f.CdiInstallNs))
+		err = controllerutil.SetOwnerReference(config, dv, scheme.Scheme)
+		Expect(err).ToNot(HaveOccurred())
+
+		By(fmt.Sprintf("Create new datavolume %s", dv.Name))
+		dv, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, ns, dv)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(dv.Annotations[controller.AnnDeleteAfterCompletion]).To(Equal("true"))
+
+		By("Verify pvc was created")
+		pvc, err := utils.WaitForPVC(f.K8sClient, ns, dv.Name)
+		Expect(err).ToNot(HaveOccurred())
+		f.ForceBindIfWaitForFirstConsumer(pvc)
+
+		By("Wait for DV to be garbage collected")
+		Eventually(func() bool {
+			_, err = f.CdiClient.CdiV1beta1().DataVolumes(ns).Get(context.TODO(), dv.Name, metav1.GetOptions{})
+			return k8serrors.IsNotFound(err)
+		}, timeout, pollingInterval).Should(BeTrue())
+
+		By("Verify PVC still exists")
```

Comment:
WDYT about verifying content in your tests as well similarly to `By("Verify content")`
throughout the importer suite

Reply:
Good idea

Reply:
done